server/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "babel app -d build",
     "prod": "npm run build; node build/server.js"
   },
-  "license": "ISC",
+  "private": true,
```

Comment:
fyi this is such a silly change but because this is not open source i changed it to private: true